client/src/document/organisms/sidebar/index.tsx

```diff
@@ -28,6 +28,16 @@ export function SidebarContainer({ doc, children }) {
     }
   }, [isSidebarOpen]);
 
+  useEffect(() => {
+    const sidebar = document.querySelector(".sidebar");
+    const currentSidebarItem = sidebar?.querySelector("em");
+    if (sidebar && currentSidebarItem) {
+      sidebar.scrollTo({
+        top: currentSidebarItem.offsetTop - window.innerHeight / 3,
+      });
```

Comment:
This won't scroll on mobile, because the `.sidebar-inner` element becomes the one which scrolls, not `.sidebar`. Something like this is a bit ugly, but works:

```suggestion
      [sidebar, sidebar.querySelector(".sidebar-inner")].forEach(n => n?.scrollTo({
        top: currentSidebarItem.offsetTop - window.innerHeight / 3,
      }));
```

(note this only fires on component load and unload, not when we resize screen - this is fine, but means you need to refresh between testing a mobile viewport and a desktop viewport)